<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registrarLocal</class>
 <widget class="QDialog" name="registrarLocal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registrar Local de Vacunación</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>651</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>421</width>
      <height>149</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Dirección de Local:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTextEdit" name="direccionLocalTxt"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Estado del Local:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="habilitarLocalRb">
         <property name="text">
          <string>Habilitado</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="inhabilitarLocalRb">
         <property name="text">
          <string>Inhabilitado</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Nombre del Local:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nombreLocalTxt"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="registrarLocalPb">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Registrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelarLocalPb">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>250</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>nombreLocalTxt</tabstop>
  <tabstop>direccionLocalTxt</tabstop>
  <tabstop>habilitarLocalRb</tabstop>
  <tabstop>inhabilitarLocalRb</tabstop>
  <tabstop>registrarLocalPb</tabstop>
  <tabstop>cancelarLocalPb</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
